src/tp268.jl

```diff
@@ -33,5 +33,5 @@ function tp268(args...)
   @constraint(nls,  8 * x[1] -      x[2] + 2 * x[3] + 5 * x[4] - 3 * x[5] - 11 ≥ 0)
   @constraint(nls, -4 * x[1] -  2 * x[2] + 3 * x[3] - 5 * x[4] +     x[5] + 30 ≥ 0)
```

Comment:
Do these have to become `@NLconstraint` or not? Ideally, it would be better to detect linear constraints properly.

Reply:
I didn't modified them in `OptimizationProblems.jl`, I think we should keep @constraint. We will detect linear constraints properly in the future with `NLPModelsJuMP`, one step after another.